openmc/mesh.py

```diff
@@ -181,6 +183,21 @@ def __repr__(self):
         string += '{0: <16}{1}{2}\n'.format('\tPixels', '=\t', self._width)
         return string
 
+    def cell_generator(self):
+        """Generator function to traverse through every [i,j,k] index
+        of the mesh.
+
```

Comment:
This is great! I've thought that we needed something like this for a long time. Could you add an "Examples" block to this docstring to illustrate how one can use this method?